py/pysparkling/initializer.py

```diff
@@ -18,8 +20,11 @@ def load_sparkling_jar(spark_context):
 
     @staticmethod
     def __add_sparkling_jar_to_spark(sc):
+        jsc = sc._jsc
+        jvm = sc._jvm
         # Add Sparkling water assembly JAR to driver
-        url = sc._jvm.java.net.URL("file://"+Initializer.__get_sw_jar())
+        sw_jar_file = Initializer.__get_sw_jar()
+        url = sc._jvm.java.net.URL("file://{}".format(sw_jar_file))
```

Comment:
`jvm.java.net.URL("file://{}".format(sw_jar_file))` would be shorter since we have already defined `jvm` above it